wvd-sh/WVD scaling script/basicScale-MSI.ps1

```diff
@@ -312,18 +293,27 @@ if ($hostpoolInfo.LoadBalancerType -eq "DepthFirst") {
                 if ($numberOfRunningHost -lt $MinimumNumberOfRDSH) {
                     $hostofsessions = $sessionHost.Sessions
                     if ($hostpoolMaxSessionLimit -ne $hostofsessions) {
-                        if ($sessionhost.Status -eq "NoHeartbeat") {
-                            $sessionhostname = $sessionhost.sessionhostname
+                        #Check the session host status and session host is healthy or not before starting the host
+                        if ($sessionhost.Status -eq "NoHeartbeat" -and $sessionhost.UpdateState -eq "Succeeded") {
+                            $sessionhostname = $sessionhost.sessionhostname | out-string
+                                $VMName = $sessionhostname.Split(".")[0]
+                                $VmInfo = Get-AzureRmVM | Where-Object { $_.Name -eq $VMName }
+                                #Check the Session host is in Maintainance
+                                if($VmInfo.Tags.Keys -contains  $maintainanceTagName){
+                                Write-Log 1 "Session Host is in Maintainace: $sessionhostname"
```

Comment:
replace "Maintainace" with "maintenance"

Reply:
We have replaced